toolkit/imageconfigs/linuxguard-amd64.yaml

```diff
@@ -201,20 +181,17 @@ scripts:
     - path: scripts/linuxguard/duid-type-to-link-layer.sh
     # Disable unused SELinux policy modules and configure SELinux policy for CI
     - path: scripts/linuxguard/selinux-ci-config.sh
+    # Setup notation cli for OCI signature verification
```

Comment:
I dont think you want this comment here. The line that it refers to is not included anyway.

Reply:
comment removed